pkg/constants/constants.go

```diff
@@ -140,3 +140,7 @@ const (
 const (
 	DefaultKlusterletPriorityClassName = "klusterlet-critical"
 )
+
+const (
+	GlobalKlusterletConfigName = "global"
```

Comment:
I kind of want to keep global as part of import-controller because merging klusterletconfig seems essential, but the rule that we hold a special klusterletconfig named "global" could be modified.